tasks/main.yml

```diff
@@ -3,6 +3,9 @@
   setup:
 
 - include_tasks: "setup-{{ ansible_os_family|lower }}.yml"
+  when: ansible_os_family.lower() == 'archlinux'
```

Comment:
There are multiple possible values for Archlinux (see https://github.com/cawilliamson/ansible-gpdpocket/blob/master/roles/iso/tasks/main.yml#L3 as an example) so I think this way of splitting between APT and Arch based distributions may make sense...

Reply:
At the moment the README and the role metadata clearly states that this role should work on Ubuntu 18.04 and Archlinux. I currently don't have any plans to support Linux Mint & Kali since I can't test this distributions. So I wouldn't care for something that isn't supported anyways :wink:

Reply:
If we want to add Debian support then it would be ok for me to change this `include_tasks` a little bit e.g.:

```
- include_tasks: "setup-{{ ansible_distribution|lower }}.yml"
```
and then have files called `setup-(ubuntu|debian|archlinux).yml`. That should be good enough for now IMHO.

Reply:
`ansible_distribution == 'archlinux'` isn't needed BTW. The task

- include_tasks: ""setup-{{ ansible_distribution|lower }}.yml""

is sufficient. You just need the include files accordingly e.g.:

```
tasks/setup-archlinux.yml
tasks/setup-ubuntu.yml
tasks/setup-debian.yml
```
It's basically the same as it is now just with the difference that `setup-debian.yml` will then really install the setup for Debian and not Ubuntu :wink: My fault was to use `ansible_os_family` variable instead of `ansible_distribution`. Because the last one will correctly print the distribution and not the OS family which is the same for Debian and Ubuntu of course.

Reply:
Applied this suggestions. It's cleaner and for arch derivates a user could always override ansible_distribution in inventory :).